packaging/create_rpm

```diff
@@ -28,6 +29,7 @@ MANAGER_RESOURCES_BASE = os.path.join(
     RPM_BUILDER_BASE,
     'cloudify/sources',
 )
+LOCAL_PACKAGES_DIR = os.path.expanduser('~/cloudify-manager-install-rpms')
```

Comment:
Do we maybe want to make it configurable with a default value?

Reply:
I'd rather have it a hard-coded path relative to the build script, but the way we're using `/tmp` to do the whole build would seem to make that awkward